unitary/examples/quantum_rpg/qaracter.py

```diff
@@ -93,44 +93,49 @@ def add_quantum_effect(self, effect, quantum_name: Union[int, str]):
         hp_obj = self.get_hp(quantum_name)
         effect(hp_obj)
 
+    @property
+    def damage(self) -> int:
+        """Returns the number of qubits measured to be zero."""
+        return sum(status == 0 for status in self.health_status.values())
+
+    @property
+    def virtue(self) -> int:
+        """Returns the number of qubits measured to be one."""
+        return sum(status == 1 for status in self.health_status.values())
+
     def is_down(self) -> bool:
         """Returns True if over half of the HP have been measured as `HURT`."""
-        damage = len(
-            [q for q in self.health_status if self.health_status[q] == 0])
-        return damage > (self.level / 2)
+        return self.damage > (self.level / 2)
 
     def is_escaped(self) -> bool:
         """Returns True if all HP are measured and at least half are `HEALTHY`."""
-        virtue = len(
-            [q for q in self.health_status if self.health_status[q] == 1])
         return len(
-            self.health_status) == self.level and virtue > (self.level / 2)
+            self.health_status) == self.level and self.virtue > (self.level / 2)
 
     def is_inactive(self) -> bool:
```

Comment:
optional nit. Would it be more readable to have an `is_active` method and `not` it as needed rather than possibly having to do `not is_inactive` for checking if a qaracter is active.

Reply:
Done.  Remove double negative and changed to is_active().